.chloggen/metrics_builder_constructor_ms_to_mbc.yaml

```
@@ -0,0 +1,20 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement # breaking?
```

Not breaking, all the APIs are in internal packages
```suggestion
change_type: enhancement
```